### PR TITLE
Feature/cms ajax form loader

### PR DESCRIPTION
**Pull request by pjosh**
* All sections loads ~the forms in ajax~ with Turbolinks
* Moving the javacript code to the `<head>` (Turbolinks best practices)
* Fixes in the field association
* Add backbone routes for the Partners section
* Add the RSS import to the documentation

### CODE REVIEW

File: backend/app/assets/javascripts/backend/views/index_items_view.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    events: {
+      'click .c-index-item': '_onClickHighlightItem',
+      'click .js-create-new': '_removeHighlight'
+    },
+
+    options: {
+      linkerTriggerClass: "js-linker",
+      selectedClass: "-selected"
+    },
```
Comment (dhakelila) [inline backend/app/assets/javascripts/backend/views/index_items_view.js]: I really like this parametrization. Have no done it before. I'll keep the idea!

**Comment by simaob** [comment]
@dhakelila Let's hold off on merging this for now! We are trying to get something similar with turbolinks, which keeps the readable and shareable URLs!

**Comment by dhakelila** [comment]
I was just about to ask why we have decide to change how to load those forms.

**Comment by simaob** [comment]
It turns out we weren't using turbolinks on the backend, so we were doing full page reloads. We will be adding it, and Jorge will just check how to maintain the central pane scroll position. So that it is not scrolled to the top.

**Comment by pjosh** [comment]
@simaob @dhakelila now we have the Turbolinks running!
In the `index_items_view.js` we have the events code to recover the index block state.

**Comment by simaob** [comment]
Thanks @pjosh nice work! And sorry for the confusion on my end!

**Comment by simaob** [comment]
Looks great to me. Well done =)

**Comment by simaob** [comment]
Maybe I'd just suggest getting rid of the `_form.html.erb` partials, as you had before.

On merge I'll squash the commits so that we don't keep the ones that are not necessary!

**Comment by pjosh** [comment]
Done @simaob !

**Pull request merged**